packages/core/src/utils/entry-points.ts

```diff
@@ -59,25 +59,29 @@ export const getPackageEntryPoints = async (
     fs,
   });
 
-  return entryPaths.map((entryPath) => {
-    const extension = path.extname(entryPath);
-    const isDefaultEntry = entryPath === defaultEntry;
-    const entryName = isDefaultEntry
-      ? 'dist'
-      : path.relative(otherEntries, entryPath).replace(extension, '');
-    const outputDir = path.join(packageRoot, entryName);
+  return entryPaths.map((relativeEntryPath) => {
+    const extension = path.extname(relativeEntryPath);
+    const isDefaultEntry = relativeEntryPath === defaultEntry;
+
+    const entryPath = path.join(packageRoot, relativeEntryPath);
+    const entryFileName = isDefaultEntry
+      ? 'index'
+      : path.relative(otherEntries, relativeEntryPath).replace(extension, '');
```

Comment:
Could `path.basename` be used here instead of string replacement?

Reply:
The entry path can be several folder deep. For example:

https://github.com/seek-oss/crackle/blob/36dcd25aa7fb36e43e44107242bfaebb29f26b01/packages/core/src/utils/entry-points.test.ts#L17

which becomes

https://github.com/seek-oss/crackle/blob/36dcd25aa7fb36e43e44107242bfaebb29f26b01/packages/core/src/utils/entry-points.test.ts#L48

Reply:
Oh right, you want everything but the extension, not just the file name.